src/main/java/edu/umich/tl/Utils.java

```diff
@@ -68,5 +70,13 @@ public static InputStream getInputStreamFromURL(String fileURL) {
 	public static boolean isEmpty(String value) {
 		return (value == null) || (value.trim().equals(""));
 	}
+	// iso8601String= "2015-01-08T23:00:00Z"
+	public static DateTime changeToDate(String iso8601String) {
```

Comment:
Well, The dateString canvas is sending is ISO8601 and all date are in the format as stated in the comment. i looked up what is ISO standard,  what i understood is if time has 'Z' then it must be in UTC( look up from WIKI `If the time is in UTC, add a Z directly after the time without a space`), To answer your question all datesString are processed in UTC from canvas and i an just converting them in to UTC Date.